angularSTEP/src/app/recipe-page/recipe-page.component.html

```diff
@@ -0,0 +1,27 @@
+<p>recipe-page works!</p>
```

Comment:
My thinking was that one of the things people would want to first know about when making a food that they might eat is whether or not they can actually eat it and what came to mind first was allergies. I made ingredients the first tab so that it would be easy to scan the list and see if there are allergens or any foods the user doesn't eat, especially since we don't have any tags as of yet and even if we did these tags might not cover every case for a person. By putting the ingredients first I aimed to make it easier for the user to be able to decide if the recipe is for them.

I have little to no experience in cooking and baking, but from the myriad of cooking and baking shows or videos I've watched through the years what tends to come first is prep, so from this standpoint having a clear list of what ingredients and tools/equipment you may need before a list of instructions to follow makes sense. While the difference may just be moving the mouse and a click or two, I still feel that the order of ingredients, tools, instructions, and extra information/blog would be most beneficial to a user.